src/ring_theory/perfection.lean

```diff
@@ -485,7 +485,9 @@ dif_neg $ λ ⟨n, hn⟩, hn rfl
 lemma val_aux_one : val_aux K v O hv p 1 = 1 :=
 (val_aux_eq $ show coeff (mod_p K v O hv p) p 0 1 ≠ 0, from one_ne_zero).trans $
 by { rw [pow_zero, pow_one, ring_hom.map_one, ← (ideal.quotient.mk _).map_one, mod_p.pre_val_mk,
-    ring_hom.map_one, v.map_one], exact @one_ne_zero (mod_p K v O hv p) _ _ }
+    ring_hom.map_one, v.map_one],
+  haveI := ne_zero.one (mod_p K v O hv p), -- why can't find this instance?
+  exact one_ne_zero }
```

Comment:
Do you want to sort this out before merging?

Reply:
I replaced it with `change ...`.